python/idsse_common/idsse/common/aws_utils.py

```diff
@@ -65,20 +59,20 @@ def aws_ls(self, path: str, prepend_path: bool = True) -> Sequence[str]:
             return [os.path.join(path, filename.split(' ')[-1]) for filename in commands_result]
         return [filename.split(' ')[-1] for filename in commands_result]
 
-    def aws_cp(self,
-               path: str,
+    @staticmethod
+    def aws_cp(path: str,
```

Comment:
See my suggestion in adding abstract 'cp' to protocal_utils.py, and rename this to just 'cp' like 'ls' above.